src/OpenLoco/Window.h

```diff
@@ -34,18 +34,18 @@ namespace OpenLoco::Ui
         wt_4,
         wt_5,
         wt_6,
-        wt_7,
-        wt_8,
-        wt_9,
-        wt_10,
-        wt_11,
+        toolbarTab = 7,
+        tab = 8,
+        buttonWithImage = 9,
+        buttonWithColour = 10,
+        button = 11,
         wt_12,
         wt_13,
-        wt_14,
+        buttonGridSort = 14,
         wt_15,
-        groupbox,
-        wt_17,
-        wt_18,
+        groupbox = 16,
+        textbox = 17,
+        dropdown = 18,
```

Comment:
I'm not sure this is dropdown its used by stepper widgets predominantly. Is there perhaps another name we could use.

Reply:
I think its just an inputBox

Reply:
You're right, it's not technically the dropdown, its the box which displays the value currently held in a dropdown, stepper widget, or other control. I don't know the name of this kind of control. It's a read-only textbox, or if I had to name it it would be a label-box. The closest analogue I can find in WinForms is a combo box, but a combo box is also just a dropdown, it doesn't encompass stepper widgets or anything else.

So I guess my ideas are:
1. labelbox
2. readonlyTextbox
3. comboBox
4. valueBox

I'm open to any other ideas

Reply:
The reason i suggested input box is because it's a user modifiable value (via stepper buttons, drop down button, or other). I think value box encompasses it's uses

Reply:
@AaronVanGeffen and myself think comboBox would be best for wt_18. Could you also add a note saying "interaction events forwarded to next widget"